functionnal_tests/cypress/integration/app_file/file_header_spec.js

```diff
@@ -0,0 +1,47 @@
+import { PAGES as p } from '../../support/urls_commands'
+
+context('The app file', function () {
+  let pdfId
+  const pdfFilename = 'the_pdf.pdf'
+  const pdfName = 'the_pdf'
+  const pdfType = 'application/pdf'
+
+  let videoId
+  const videoFilename = 'video.mp4'
+  const videoName = 'video'
+  const videoType = 'video/mp4'
+
+  before(function () {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.loginAs('administrators')
+    cy.fixture('baseWorkspace').then(workspace => {
+      cy.createFile(pdfFilename, pdfType, pdfName, workspace.workspace_id).then(file => pdfId = file.content_id)
+      cy.createFile(videoFilename, videoType, videoName, workspace.workspace_id).then(file => videoId = file.content_id)
+    })
+  })
+
+
+  beforeEach(function () {
+    cy.loginAs('administrators')
+  })
+
+
+  afterEach(function () {
+    cy.cancelXHR()
+  })
+
+  describe('when type is PDF', () => {
```

Comment:
I think you can wrap your 2 describes into a global one like 'specific button display' or 'content specific action' or something

Reply:
Fixed